pyproject.toml

```
@@ -55,6 +55,7 @@ docs = [
 "flexpart.cosmo" = "aiida_flexpart.calculations.flexpart_cosmo:FlexpartCosmoCalculation"
 "flexpart.ifs" = "aiida_flexpart.calculations.flexpart_ifs:FlexpartIfsCalculation"
 "flexpart.post" = "aiida_flexpart.calculations.flexpart_post:PostProcessingCalculation"
+"collect.sens" = "aiida_flexpart.calculations.collect_sens:CollectSensCalculation"
```

also here, please rename:

```suggestion
"collect.sensitivities" = "aiida_flexpart.calculations.collect_sens:CollectSensitivitiesCalculation"
```